Source/common/SNTXPCControlInterface.m

```diff
@@ -27,10 +27,16 @@
 @implementation SNTXPCControlInterface
 
 + (NSString *)serviceID {
+#ifdef SANTAADHOC
+  // The mach service for an adhoc signed ES sysx uses the "endpoint-security" prefix instead of
+  // the teamid. In Santa's case it will be endpoint-security.com.google.santa.daemon.xpc.
+  return [NSString stringWithFormat:@"endpoint-security.%@.xpc", kBundleID];
```

Comment:
We may want to consider setting `NSEndpointSecurityMachServiceName` (see man page for `EndpointSecurity(7)`. This should mitigate the need for different mach service names.

Reply:
Something to note tho - this key didn't exist in 10.15, it came a bit later (10.15.4 maybe?)

Reply:
Thanks! Let's wait to adopt `NSEndpointSecurityMachServiceName` until we drop support for 10.15.